nextcloudappstore/core/user/views.py

```diff
@@ -0,0 +1,19 @@
+from django.contrib import messages
+from django.shortcuts import render
+from django.utils.translation import ugettext as _
+
+import registration.backends.simple.views 
+
+from nextcloudappstore.core.user.forms import RegistrationFormRecaptcha
+
+
+class RegistrationView(registration.backends.simple.views.RegistrationView):
+    
+    form_class = RegistrationFormRecaptcha
+
+    def get_success_url(self, user):
+        return u'/'
```

Comment:
Should already be unicode in Python 3 right? Example maybe used Python 2? Am I wrong?